tools/clang/test/HLSL/workgraph/node_compute_compatibility.hlsl

```diff
@@ -22,40 +23,79 @@ void node01()
 [Shader("compute")]             // expected-note {{compute defined here}}
 [NumThreads(128,1,1)]
 [NodeLaunch("Coalescing")]      // expected-error {{Node shader 'node02' with coalescing launch type is not compatible with compute}}
-void node02(GroupNodeInputRecords<RECORD> input)
+void node02()
```

Comment:
I know clang-format doesn't play well with the diagnostic verifier lines, but it would be nice to format the code here more LLVM-style for consistency.

Reply:
I've removed the unnecessary spaces.